rosdep/python.yaml

```
@@ -4631,6 +4631,8 @@ python3-gitlab:
     xenial:
       pip:
         packages: [python-gitlab]
+python3-lark-parser:
+  ubuntu: [python3-lark-parser]
```

```suggestion
  ubuntu:
    bionic: [python3-lark-parser]
```

This package is only available in bionic.

---

@tfoote suggested to add it as is. I don't mind either or.